src/uct/ugni/rdma/ugni_rdma_ep.c

```diff
@@ -607,10 +607,10 @@ static ucs_status_t uct_ugni_ep_get_composed_fma_rdma(uct_ep_h tl_ep, void *buff
     rdma_length = length - iface->config.fma_seg_size;
     fma_length = iface->config.fma_seg_size;
 
-    UCT_TL_IFACE_GET_TX_DESC(&iface->super, iface->free_desc_get_buffer,
+    UCT_TL_IFACE_GET_TX_DESC(&iface->super.super, iface->free_desc_get_buffer,
```

Comment:
looks a bit ugly super.super. Maybe we should macro it or just define another variable ?